packages/website/ts/components/ui/account_connection.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+import { Circle } from 'ts/components/ui/circle';
+import { Container } from 'ts/components/ui/container';
+import { Text } from 'ts/components/ui/text';
+import { colors } from 'ts/style/colors';
+import { AccountState } from 'ts/types';
+
+export interface AccountConnectionProps {
+    accountState: AccountState;
+    injectedProviderName: string;
+}
+
+export const AccountConnection: React.StatelessComponent<AccountConnectionProps> = ({
+    accountState,
+    injectedProviderName,
+}) => {
+    return (
+        <Container className="flex items-center">
+            <Circle diameter={6} fillColor={getInjectedProviderColor(accountState)} />
+            <Container marginLeft="6px">
+                <Text fontSize="12px" lineHeight="14px" fontColor={colors.darkGrey}>
+                    {injectedProviderName}
+                </Text>
+            </Container>
+        </Container>
+    );
+};
+
+function getInjectedProviderColor(accountState: AccountState): string {
```

Comment:
So the question of free-form `function getInjectedProvider(){}` vs. `const getInjectedProvider = () => {}` is weird. I feel like free-form `function` is ES5 and we work in a ES6/Typescript environment, so we should always use arrow functions? Like in ES5 `function` is used to define "classes" since it has a `prototype` and a `this` property whereas arrow functions do not, and are intended just to be functions. 

This doesn't matter really just sharing some thoughts.